homeassistant/components/kitchen_sink/config_flow.py

```diff
@@ -0,0 +1,19 @@
+"""Config flow to configure the Kitchen Sink component."""
```

Comment:
Just wondering: do we need specific config-flow tests?
Or is it sufficient that it is covered via existing `test_init.py`?

Reply:
I'll add a specific test for clarity